addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/handler/YamahaZoneThingHandler.java

```diff
@@ -82,10 +68,12 @@
     private ServiceRegistration<?> servicePreset;
     private ServiceRegistration<?> serviceAvailableInputs;
 
-    /// State
+    /// Invalidateable
```

Comment:
This is the technical description. It was meant to be for new developers to see where all internal states should be grouped. Maybe call it "Internal state" instead?